common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -212,9 +206,11 @@ export interface IBranchOrigin {
  */
 export interface ISequencedDocumentMessage {
 	/**
-	 * The client ID that submitted the delta.
+	 * The client ID that submitted the message.
+	 * For server generated messages the clientId will be null;
 	 */
-	clientId: string;
+	// eslint-disable-next-line @rushstack/no-new-null
+	clientId: string | null;
```

Comment:
The lint suppression answers my question about whether this was intentional, but why is null correct here?